File: 3790-fruits-into-baskets-ii/fruits-into-baskets-ii.cpp
class Solution {
public:
    int numOfUnplacedFruits(vector<int>& fruits, vector<int>& baskets) {
        int res=0;
        vector<int>seen;
        for(int f:fruits){
            int flag=0;
            for(int i=0;i<baskets.size();i++){
                if((f<=baskets[i])&&(find(seen.begin(),seen.end(),i)==seen.end())){
                    seen.push_back(i);
                    flag=1;
                    break;
                }
            }
            if(flag==0){
                res++;
            }
        }
        return res;
        
    }
};

File: 3790-fruits-into-baskets-ii/fruits-into-baskets-ii.py
class Solution:
    def numOfUnplacedFruits(self, fruits: List[int], baskets: List[int]) -> int:
        count=0
        seen=set()
        for f in fruits:
            flag=False
            for j in range(len(baskets)):
                if j not in seen and f<=baskets[j]:
                    seen.add(j)
                    flag=True
                    break
            if not flag:
                count+=1
        return count